src/main/scala/strawman/collection/mutable/Set.scala

```diff
@@ -5,13 +5,16 @@ import strawman.collection.IterableOnce
 import scala.{Int, Boolean, Unit, Option, Some, None}
 import scala.Predef.???
 
-trait Set[A] extends collection.Set[A] with Growable[A] {
+/** Base trait for mutable sets */
+trait Set[A]
+  extends collection.Set[A]
+    with collection.SetLike[A, Set]
+    with Growable[A] {
 
   def +=(elem: A): this.type
   def -=(elem: A): this.type
 
-  def contains(elem: A): Boolean
-  def get(elem: A): Option[A]
+  def get(elem: A): Option[A] = if (contains(elem)) Some(elem) else None
```

Comment:
@szeiger I agree, implementation is wrong. I wanted it to be the same as `get` on a Java set.